crates/re_space_view_spatial/src/contexts/mod.rs

```diff
@@ -23,6 +23,7 @@ use re_viewer_context::{
 /// Context objects for a single entity in a spatial scene.
 pub struct SpatialSceneEntityContext<'a> {
     pub world_from_entity: glam::Affine3A,
+    pub pinhole_scale: Option<glam::Vec2>,
```

Comment:
This is likely a bad naming since (most) pinhole scaling is handled elsewhere already. Maybe `radii_scale_factor`?

Reply:
Nvm, I renamed everything in ac2c0cfa5ba17273d1e942fe9c4740ddd38cc9fa to be clearer on the intent.